apps/web/lib/apps/installation/[[...step]]/getServerSideProps.ts

```diff
@@ -307,7 +306,6 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
 
   return {
     props: {
-      ...(await serverSideTranslations(locale, ["common"])),
```

Comment:
`serverSideTranslations` from `next-i18next` uses React.createContext and hence cannot be used server-side with Turbopack, because Turbopack is very strict with client code sneaking in server-side

Reply:
and returning `await serverSideTranslations` in `getServerSideProps` files is not needed anymore in App Router